service/src/test/java/bio/terra/pipelines/dependencies/leonardo/AppUtilsTest.java

```diff
@@ -0,0 +1,611 @@
+                            "machineType": "Standard_A2_v2",
+                            "autoscalingEnabled": false
+                        },
+                        "errors": [],
+                        "status": "RUNNING",
+                        "proxyUrls": {
+                            "wds": "https://lzblahblahblah.servicebus.windows.net/wds-${workspaceId}/wds"
+                        },
+                        "appName": "wds-${workspaceId}",
+                        "appType": "WDS",
+                        "diskName": null,
+                        "auditInfo": {
+                            "creator": "me@broadinstitute.org",
+                            "createdDate": "2022-10-10T16:01:36.660590Z",
+                            "destroyedDate": null,
+                            "dateAccessed": "2023-02-09T16:01:36.660590Z"
+                        },
+                        "accessScope": null,
+                        "labels": {}
+                    }""",
+                Map.of("workspaceId", workspaceId)));
+
+    separatedWdsAppNotWdsAppName =
+        ListAppResponse.fromJson(
+            StringSubstitutor.replace(
+                """
+                                {
+                                    "workspaceId": "${workspaceId}",
+                                    "cloudContext": {
+                                        "cloudProvider": "AZURE",
+                                        "cloudResource": "blah-blah-blah"
+                                    },
+                                    "kubernetesRuntimeConfig": {
+                                        "numNodes": 1,
+                                        "machineType": "Standard_A2_v2",
+                                        "autoscalingEnabled": false
+                                    },
+                                    "errors": [],
+                                    "status": "RUNNING",
+                                    "proxyUrls": {
+                                        "wds": "https://lzblahblahblah.shouldneverwork.servicebus.windows.net/wds-${workspaceId}/wds"
+                                    },
+                                    "appName": "notwds-${workspaceId}",
+                                    "appType": "WDS",
+                                    "diskName": null,
+                                    "auditInfo": {
+                                        "creator": "me@broadinstitute.org",
+                                        "createdDate": "2022-10-10T16:01:36.660590Z",
+                                        "destroyedDate": null,
+                                        "dateAccessed": "2023-02-09T16:01:36.660590Z"
+                                    },
+                                    "accessScope": null,
+                                    "labels": {}
+                                }""",
+                Map.of("workspaceId", workspaceId)));
+
+    cromwellListAppResponse =
```

Comment:
should any of these be renamed to reference cbas rather than cromwell? or no because cbas is still a CROMWELL appType?

Reply:
yea i think we're reference the appType here as opposed to what proxyUrl we're grabbing